src/main/java/koreatech/in/controller/OwnerController.java

```diff
@@ -171,7 +171,11 @@ ResponseEntity<EmptyResponse> register(@RequestBody @Valid OwnerRegisterRequest
             throw new BaseException(ExceptionInformation.REQUEST_DATA_INVALID);
         }
 
-        ownerService.register(request);
+        if (request.getShopId() == null) {
+            ownerService.regesterWithShop(request);
```

Comment:
```suggestion
            ownerService.registerWithShop(request);
```

오타 있어요!

IntelliJ SpellChecker 이용하면 오타정정할 수 있으니 참고해보세요~ (이미 깔려있을수도 있음..)